topology/sof-apl-hda.m4

```diff
@@ -0,0 +1,92 @@
+#
+# Topology for generic Apollolake board with no codec.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`ssp.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> iDisp1
+# PCM1 ----> Volume <----- iDisp2
```

Comment:
fix arrow direction

Reply:
Okay, I will fix this in v3. I also need to revise the PCM name for HDMI from Portx to HDMIx, so that the generic DSP HDA driver can find the HDMI links by checking the link name and do proper inItialization for HDMI.